### DISCUSSION TITLE
Sprawy organizacyjne - czy leci z nami pilot?

### DISCUSSION THREAD

**Discussion by AroKol78** [MajkiIT/polish-ads-filter#20273] [General] [answered]
@MajkiIT / @olegwukr 
zauważyłem, że przez ostatnie miesiące nastąpił drastyczny spadek aktywności list:
- POL: Oficjalne Polskie Filtry do AdBlocka, uBlocka Origin i AdGuarda
- POL: Oficjalne polskie filtry przeciwko alertom o Adblocku

mam nadzieję, że obecna sytuacja jest przejściowa i wkrótce się poprawi.

**Comment by xxcriticxx** [comment, marked as answer]
cc @hawkeye116477 @krystian3w

**Reply by hawkeye116477** [reply]
Co mnie wołasz? Ja nic nie wiem, ja tu tylko sprzątam.... Jesteś zastępcą szeryfa Majkela i jego prawą ręką, to chyba raczej ty powinieneś coś wiedzieć.

Ja tylko mogę zgadnąć, że może teraz zbiera kasę na PS5 Pro albo nową kartę graficzną albo jakiś TV NanoLED/MicroLED i nie ma czasu na darmowe aktualizacje listy.
A robi coś 1-2 razy na miesiąc, bo nie chce żeby gorhill wypisał listę z uBO póki nie znalazł godnego następcy.

**Reply by xxcriticxx** [reply]
@hawkeye116477 to ja chyba tobie kupie nowa miotle na amazonie

**Reply by AroKol78** [reply]
po mojemu jedna osoba z uprawnieniami to za mało (wczasy,inne sprawy życiowe,itp).
ktoś jeszcze powinien dostać większe uprawnienia
choćby po to aby na bieżąco można było skorygować błędne/krytyczne filtry, czy obsłużyć zgłoszenie w rozsądnym czasie.

**Reply by hawkeye116477** [reply]
Tyle, że oprócz Majkela, są inne osoby z mniejszymi uprawnieniam, np. właśnie xxcriticxx (który raczej nie zajmuje się pisaniem filtrów, jedynie akceptuje PR) i chyba też F4z/AdamWR ma jeszcze rangę **collaborator**, ale on z kolei rzadko coś tu wrzuca, najczęściej pracuje dla zespołu AG. A admin jest tylko jeden, bo jeżeli repo jest podpięte pod konto osobiste, to 1 to max, więcej jest tylko dla organizacji, czyli np [PolishFiltersTeam](https://github.com/PolishFiltersTeam/) :smile:.

Czyli ogólnie podsumowując, bieda, bo pomimo, że niby oprócz Majkela są ludzie, którzy mogą coś tu robić, nie muszą się pytać o zgodę, bo mają odpowiednie uprawnienia, to rzadko coś robią albo jedynie zatwierdzają wszystko, bo nie mają czasu.

**Reply by hawkeye116477** [reply]
W skrócie tak to się prezentuje w firmie **Majkel zadziała i ty też możesz**....
_MajkiIT_ - prezes firmy (aktualnie nieobecny z powodu spotkania biznesowego z firmą Sony)
_xxcriticxx_ - starszy specjalista od zatwierdzania propozycji zmian, tzw. Pull Request (i zakupów w Amazonie)
_AdamWr/F4z_ - starszy specjalista od list filtrów (obecnie na emeryturze)
_MajkiIT-automat_ - robot, który wypieka chleb i zastąpi też wkrótce większość pracowników, z uwagi na to, że jest najaktywniejszy

**Reply by xxcriticxx** [reply]
@hawkeye116477 dobre podsumowanie malo sie w portki nie posikalem

**Reply by hawkeye116477** [reply]
No niestety tak to wygląda, brakuje ludzi do zarządzania listą reklamową. Jedyną zaletą jest to, że chociaż jest zatwierdzający Pull Requesty, bo tak to byłaby większa bieda.

**Reply by xxcriticxx** [reply]
czekam na wasze PR

**Reply by krystian3w** [reply]
Może też przydałoby się ograniczyć sekcje z jakiś 27 do mniejszej ilości.

**Reply by xxcriticxx** [reply]
sekcje czego @krystian3w?

**Reply by krystian3w** [reply]
sekcje listy adblock, pewnie połowa nie ogarnia takiego podziału i nie zrobi PR.

**Reply by hawkeye116477** [reply]
Z tym, że lista reklamowa ma 50 sekcji, a nie 27.

**Reply by krystian3w** [reply]
Ale kilkanaście chyba jest puste z jakiegoś powodu.

**Reply by AroKol78** [reply]
wygląda na to, że MajkiIT rzeczywiście nie ma czasu (repo "MajkiIT/html_static" aktywny 3h temu).

prawie wszystkie _.pl_ idą teraz na _AdguardFilters_ 
2x  "POL: Oficjalne Polskie..."  oby nie skończyły tak jak NanoFilters/anti-adblock-killer/...
(daleko nie trzeba szukać podobnego przykładu https://github.com/MajkiIT/polish-ads-filter/discussions/17251)

**Reply by hawkeye116477** [reply]
Zawsze jeszcze jest ten co gra w gałę ⚽ czy jak mu tam było...

**Reply by xxcriticxx** [reply]
Jak bedzie potrzeba to jest tutaj wystarczajaco osob zeby stworzyc nowa liste.  Nie przezywajcie jak stonka wykopki!

**Reply by hawkeye116477** [reply]
W takim razie kto by taką listą się zajmował? Ja nie widzę kandydatów. A PFT już ma wystarczająco list, kolejnych nie planuje. No, chyba, że @xxcriticxx we własnej osobie, ale jakoś powątpiewam w to... Osoby tutaj obecne mają inne zajęcia i raczej nie starczy im czasu lub niektórzy nie potrafią, także proszę tutaj nie ściemniać :stuck_out_tongue:.

Tak więc jedynie można sobie pomarzyć, że może nagle pojawi się ktoś nowy ze swoją listą albo będziemy musieli korzystać z listy tego piłkarza + może ekipa gorhilla dorzuci parę reguł tylko dla uBO + AdGuard Base.

**Reply by xxcriticxx** [reply]
Wiec proste listy nowej nie będzie!

**Reply by AroKol78** [reply]
jednym słowem czas kończyć flaszkę, ostatni gasi światło.

**Reply by garry-ut99** [reply]
> hawkeye116477 : https://github.com/MajkiIT/polish-ads-filter/issues/22084#issuecomment-1811546166 : albo nie ma na to czasu, bo nawet widać, że ostatnio listę dla adblockerów zaniedbał, tzn. znacznie mniej wrzuca niż w ubiegłych roku, a zgłoszenia często są zamykane przez bota z powodu braku aktywności.

Dlatego wiele zgłoszeń reklamowych ostatnio jest przekierowywanych na : 
- https://github.com/easylistpolish/easylistpolish/issues?q=sort%3Aupdated-desc i tam są naprawiane, więc dobrze tam zgłaszać 
   - i subskrybować dodatkowo `EasyList Polish` jako uzupełnienie do `Oficjalne Polskie Filtry do AdBlocka, uBlocka Origin i AdGuarda`.
- trudniejsze reklamy i anty-adblocki przekierowywać na : 
   - https://github.com/uBlockOrigin/uAssets/issues  (strony WP do https://github.com/uBlockOrigin/uAssets/issues/12214 )
   - https://github.com/AdguardTeam/AdguardFilters/issues ( uaktywnić listę `Adguard Base` )

**Comment by krystian3w** [comment]
Testowałeś już alfę Firefox 68 dla XP?

Pod Vistą niby się wywala.

**Reply by AroKol78** [reply]
>Testowałeś już alfę Firefox 68 dla XP?

nie - brak d3d9,... 
> Pod Vistą niby się wywala.

spróbuj bez multi-process(e10s)
`browser.tabs.remote.autostart` `false` i/lub `dom.ipc.processCount` `0` or `1`

powyłączaj akceleracje sprzętową - możliwy brak d3d10

**Reply by hawkeye116477** [reply]
XP to do muzeum, pora przejść na coś nowszego :smile:.

**Reply by AroKol78** [reply]
> XP to do muzeum, pora przejść na coś nowszego 😄.

jeszcze więcej mam "muzeum" C=64, A600,A1200,A4000  😄
a na telefon przeglądanie stron internetowych to istny koszmar
